backend/README.md

```diff
@@ -0,0 +1,202 @@
+# Backend
+
+## MODELS
+
+We have two models in for this app.
+
+-   USER -> This model set up the requirements for the users.
+    With enum of the three different types of users:
+    ` Enum = ['CUSTOMER', 'SUPPLIER', 'ADMIN']`
+
+To sign up user, you must
+
+`{ 'email':'a@b.c', 'password':'abc123', 'role':'enum' }`
+
+-   RATING -> This model sets the rating from 'CUSTOMER' to 'SUPPLIER' and the rating range and date.
+
+## ROUTING
+
+-   /users GET - Requires authentication, returns all users.
+-   /users/:id GET - Requires authentication, returns requested user with specified id.
+-   /users/:id/ratings GET - Requires authentication, returns all ratings for the requested user have made.
+-   /hairdressers/:id/ratings GET - No authentication, returns all ratings for the requested hairdresser.
+-   /hairdressers/:id/ratings POST - Requires authentication, create a new rating for the specified hairdresser.
+
+## API Endpoints
+
+### Public routes
+
+---
+
+### Signup
+
+Method: `POST`<br>
+Path: `/signup`
+
+#### Example of API request
+
+```js
+{
+	"email": "test@test.com",
+	"password": "password",
+	"role": "CUSTOMER",
+	"description": "Test account",
+	"address": {
+		"street": "Kungsgatan 34",
+		"city": "Stockholm",
+		"zip": 12345
+	}
+}
+```
+
+#### Fields in API request
+
+-   `email` **Required**
+-   `password` **Required**
+-   `role` **Required** - valid values are _CUSTOMER_ and _SUPPLIER_
+-   `description` **Optional** - max length 250
+-   `address` **Optional**
+    -   `street` Street address - max length 250
+    -   `city` City name - max length 250
+    -   `zip` Zip code - max length 20
+
+#### Example of API response
+
+```js
+{
+	"status": 200,
+	"token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJlbWFpbCI6Im1AbS5jb20iLCJpYXQiOjE2MjE5MzQ1NTI4NjcsImV4cCI6MTYyMTkzODE1Mjg2N30.QnAfjfydKa4-Z2aLRuNFw8co6G9Oev8TEu3Aiyt3QVE",
```

Comment:
Think about not showing sequences or other possible data that could in some form help potential hackers.

Reply:
Noted and changed.